app/components/school-session-attributes.js

```diff
@@ -1,56 +1,59 @@
-import Component from '@ember/component';
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
 import { all } from 'rsvp';
+import { tracked } from '@glimmer/tracking';
+import { dropTask, restartableTask } from 'ember-concurrency-decorators';
+
+
+export default class SchoolSessionAttributesComponent extends Component {
+  @service store;
+  @tracked showSessionAttendanceRequiredConfig;
+  @tracked showSessionSupplementalConfig;
+  @tracked showSessionSpecialAttireRequiredConfig;
+  @tracked showSessionSpecialEquipmentRequiredConfig;
+
+  @restartableTask
+  *load(element, [school]) {
+    const schoolConfigs = yield school.configurations;
+    this.showSessionAttendanceRequiredConfig = schoolConfigs.findBy('name', 'showSessionAttendanceRequired');
+    this.showSessionSupplementalConfig = schoolConfigs.findBy('name', 'showSessionSupplemental');
+    this.showSessionSpecialAttireRequiredConfig = schoolConfigs.findBy('name', 'showSessionSpecialAttireRequired');
+    this.showSessionSpecialEquipmentRequiredConfig = schoolConfigs.findBy('name', 'showSessionSpecialEquipmentRequired');
+  }
 
-export default Component.extend({
-  store: service(),
-
-  tagName: "",
-
-  canUpdate: false,
-  details: false,
-  isManaging: false,
-  school: null,
-
-  showSessionAttendanceRequired: computed('school.configurations.[]', 'school.configurations.@each.value', async function() {
-    return await this.school.getConfigValue('showSessionAttendanceRequired');
-  }),
-
-  showSessionSupplemental: computed('school.configurations.[]', 'school.configurations.@each.value', async function() {
-    return await this.school.getConfigValue('showSessionSupplemental');
-  }),
-
-  showSessionSpecialAttireRequired: computed('school.configurations.[]', 'school.configurations.@each.value', async function() {
-    return await this.school.getConfigValue('showSessionSpecialAttireRequired');
-  }),
-
-  showSessionSpecialEquipmentRequired: computed('school.configurations.[]', 'school.configurations.@each.value', async function() {
-    return await this.school.getConfigValue('showSessionSpecialEquipmentRequired');
-  }),
+  get showSessionAttendanceRequired() {
+    return JSON.parse(this.showSessionAttendanceRequiredConfig?.value ?? null);
```

Comment:
gnarly.

Reply:
Yea, agreed. Gonna add this to the model so at least it doesn't need to be repeated.